cli/Cargo.toml

```diff
@@ -63,9 +63,14 @@ webpki-roots = "0.19.0"
 walkdir = "2.3.1"
 warp = "0.2.2"
 semver-parser = "0.9.0"
+# TODO(bartlomieju): make sure we're using exactly same versions
+# of "swc_*" as dprint-plugin-typescript
+swc_common = "=0.5.9"
+swc_ecma_ast = "=0.18.1"
+swc_ecma_parser = "=0.21.6"
+swc_ecma_parser_macros = "=0.4.1"
```

Comment:
Note that I will be removing this explicit reference in dprint soon. Right now swc_ecma_parser depends on 0.4, but it really depends on 0.4.1. I just opened a PR in swc to fix this.

Reply:
It might be helpful for us if dprint re-exported the swc namespace so we could use that rather than trying to keep the cargo versions in sync.

Reply:
I was wondering if there was a better way. Opened https://github.com/dsherret/dprint/issues/164